src/session.rs

```diff
@@ -3031,8 +3041,23 @@ impl Session {
             Command::Edit(ref paths) => {
                 if paths.is_empty() {
                     self.unimplemented();
-                } else if let Err(e) = self.edit(paths) {
-                    self.message(format!("Error loading path(s): {}", e), MessageType::Error);
+                }
+
+                match self.edit(paths) {
+                    Ok((success_count, fail_count)) => {
+                        if success_count + fail_count > 0 {
```

Comment:
```suggestion
                        if success_count + fail_count > 1 {
```
Otherwise it'll always trigger :)

Reply:
Facts, missclick